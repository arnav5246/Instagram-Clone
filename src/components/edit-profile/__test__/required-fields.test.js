import React from 'react'
import { create } from 'react-test-renderer'
import RequiredInputs from '../required-inputs'

describe('Required-fields Component', () => {
  const props = {
    fields: {
      username: 'admin',
      firstname: 'iam_',
      surname: 'admin',
      email: 'admin@gmail.com',
    },
    change: jest.fn(),
  }

  it('should match snapshot', () => {
    const tree = create(<RequiredInputs {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
