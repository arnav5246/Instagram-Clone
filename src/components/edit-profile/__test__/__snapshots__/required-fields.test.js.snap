// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Required-fields Component should match snapshot 1`] = `
Array [
  <div
    className="edit_un_div"
  >
    <span
      className="edit_span"
    >
      Username
    </span>
    <input
      autoComplete="false"
      disabled={false}
      maxLength="32"
      onChange={[Function]}
      placeholder="Username"
      spellCheck="false"
      type="text"
      value="admin"
    />
  </div>,
  <div
    className="edit_un_div"
  >
    <span
      className="edit_span"
    >
      Firstname
    </span>
    <input
      autoComplete="false"
      disabled={false}
      maxLength="32"
      onChange={[Function]}
      placeholder="Firstname"
      spellCheck="false"
      type="text"
      value="iam_"
    />
  </div>,
  <div
    className="edit_un_div"
  >
    <span
      className="edit_span"
    >
      Surname
    </span>
    <input
      autoComplete="false"
      disabled={false}
      maxLength="32"
      onChange={[Function]}
      placeholder="Surname"
      spellCheck="false"
      type="text"
      value="admin"
    />
  </div>,
  <div
    className="edit_un_div"
  >
    <span
      className="edit_span"
    >
      Email
    </span>
    <input
      autoComplete="false"
      disabled={false}
      maxLength="32"
      onChange={[Function]}
      placeholder="Email"
      spellCheck="false"
      type="email"
      value="admin@gmail.com"
    />
  </div>,
]
`;
